### DISCUSSION TITLE
How to share the same store across multiple electron windows?

### DISCUSSION THREAD

**Discussion by martinszeltins** [vuejs/pinia#1961] [Help and Questions] [answered]
I am making an electron app and I have multiple pinia stores defined. However when a new electron window is opened it always creates a new store because the whole app is bootstrapped anew. Each window will instantiate its own store and each window is a separate process.

The way a I open a new window is by using Vue router and giving the window url as `http://localhost:3000/#about`.

Is there some easy way to share the same store across multiple electron windows?

**Comment by justingel** [comment]
Facing the same problem rn

**Comment by AntonDeMagic** [comment, marked as answer]
We had the same struggle with our Electron app and finally decided to build a [plugin](https://www.npmjs.com/package/electron-pinia-sync) to keep stores in sync between processes without the IPC headache. Feel free to give it a spin and let us know what you think!

Hope it helps!
